server/playerAPIsUtils.go

```diff
@@ -12,6 +12,13 @@ func getPlayerAPIs(db persistence.DB, players []model.Player) (map[model.PlayerI
 		var pAPI interaction.Player
 		pm, err := db.GetInteraction(p.ID)
 
+		for i, m := range pm.Interactions {
+			if m.Mode == interaction.NPC {
+				m.Info = &npcActionHandler{}
```

Comment:
Shouldn't we have a global var that is `*npcActionHandler` and then set that for all of the `Info`s of all NPCs ever?

Reply:
in this file?

Reply:
🤷‍♂ sure. otherwise we don't gain the benefit of having a lock on it.

Reply:
oh I see 👍